### ISSUE TITLE
Cannot create a new Caddy account

### ISSUE THREAD

**Issue by rg3h** [caddyserver/website#358]
Cannot create a new account.
When you attempt to create an account, it sends you an activation email. 
Clicking on the link in the email takes you a welcome page with an activation button, but nothing happens when you click on it.
I  tried in Brave, Chrome, Firefox on a Windows machine with the same problem.
Opening dev tools shows this error:
activate-account-d99…b8f9a330674d90.js:1 Uncaught ReferenceError: $ is not defined
    at activate-account-d99975c8d48d4589781b95e27545592a4ce70150a82803bef9b8f9a330674d90.js:1:21
    at activate-account-d99975c8d48d4589781b95e27545592a4ce70150a82803bef9b8f9a330674d90.js:4:218

On this page from the email link:
https://caddy.community/u/activate-account/efee85da7315f75be29fea31fa9334c5

**Comment by mholt** [comment]
Huh. Second report of this in a week. I wonder if Discourse has a buggy release. Looks like "$" is not defined (jQuery?)

What's your username? I'll activate for you.

**Comment by rg3h** [comment]
Hi Matt! Awesome to meet you.
My username is rg3h (email ***@***.***)
It does look like maybe jquery got left off the js includes...

[side note 1: you have two very old school googlers loving caddy; I was
"rcg" back in the day and the other person was an extremely early googler
who pointed me to caddy; he is wicked smart]

[side note 2: I am trying a convoluted thing -- cloudflare to AWS lightsail
and using caddy as the server. It's a little bonked with the http-to-https
redirection cloudflare and AWS provide, but I hope to figure it out and not
get the re-redirect loops.  I did figure out a 403 problem that was
plaguing some folks with caddy and plan to post to the community once I am
in]

Great work! I venture we might be able to video chat or something once we
are not both so busy with things. That would be fun to learn about your
past and your future plans.

**Comment by mholt** [comment]
Yeah it looks like when the forum rebuilt after the ugprade it... didn't rebuild everything. My SSH session actually froze too so I dunno what's up with that. Trying it again.

> [side note 1: you have two very old school googlers loving caddy; I was
> "rcg" back in the day and the other person was an extremely early googler
> who pointed me to caddy; he is wicked smart]

That's cool! I'd love to chat more about this when we have a chance.

As for the "convoluated" thing, yeah, we can talk about it in the forum. Or if you want to sign up for a [sponsorship](https://caddyserver.com/sponsor) we can help out in private.

And of course, I can hop on a call to chat tech plans at any point. Just book a time: https://matt.chat

**Comment by mholt** [comment]
Should be taken care of now.

**Comment by rg3h** [comment]
Hi Matt!
What is your normal email and I can catch you up on things -- I talked to
the other google (Amit) and he's happy to join in a chat sometime after
holidays.

I was able to log in, so thanks for that!

I wanted to add a comment on how I solved a 403 problem that people were
having
(RE: How to solve status 403 in Caddy version 2) but it was closed. I
posted to the last commenter, who might be a moderator.

Basically instead of usermod –aG <group> caddy
I noted that /usr/share/caddy is owned by root so I did sudo chown
root:root /var/www and /var/www/index.html
(I'm not seeing user caddy as creator or owner for the index and logs).

If there is another place or way to post, let me know!

Amit and I look forward to catching up with you and I can give more
backstory in your regular email!

rich

-----
Fixing a 403 error:
If you were able to see the default caddy index.html, but when you changed
the root location in /etc/caddy/Caddyfile and got a 403
note the owner: ls -ld /usr/share/caddy/  (where the default index.html
lives). Mine was owned by root not caddy
so I did sudo chown root:root /var/www  and sudo chown root:root
/var/www/index.html to fix the 403 problem

**Comment by francislavoie** [comment]
That sounds weird, if you're running Caddy as a systemd service then Caddy would be running as the `caddy` user. How are you running Caddy?

Anyway probably best if you open a new topic on the forums to continue this.

**Comment by mholt** [comment]
@rg3h You can send an email to the address on this page: https://caddyserver.com/support

**Comment by rg3h** [comment]
I installed caddy:  sudo apt install caddy
Then I did some Caddyfile changes and ran
sudo caddy reload --config /etc/caddy/Caddyfile
It looks like there is a system daemon running. I also do this so it runs
on reboot:

sudo systemctl enable --now caddy

I'm doing this all on AWS Lightsail using an ubuntu config.
We could spin up a vanilla instance and do the install again (esp if there
is a preferred way) and see what the permissions are.
